staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -34,7 +34,8 @@ import (
 )
 
 // DeleteResource returns a function that will handle a resource deletion
-func DeleteResource(r rest.GracefulDeleter, allowsOptions bool, scope RequestScope, admit admission.Interface) http.HandlerFunc {
+// TODO admission here becomes solely validating admission
```

Comment:
Removing `mutatingAdmission` and related code will complete this TODO, won't it?

Reply:
> Removing mutatingAdmission and related code will complete this TODO, won't it?

It would, but I don't plan for this pull to convert the admission chain.  Given the runway, I think the complete conversion of admission plugins may end up pushed to 1.10

Reply:
Do you mean this: in 1.9 our existing admission plugins that do both mutation and validation will be categorized as mutation admissions, so we still need to run mutation admission in Delete and Update?

Reply:
> Do you mean this: in 1.9 our existing admission plugins that do both mutation and validation will be categorized as mutation admissions, so we still need to run mutation admission in Delete and Update?

existing admission plugins that do both mutation and validation **may** be categorized as mutation admissions.

I plan to go after a few to make sure this new wiring works correctly.